latex/epoch.tex

```diff
@@ -314,25 +398,85 @@ \subsection{Rewards Distribution}
       & ~~~\to (\HashKey \mapsto \Distr)
       & \text{pool individual reward} \\
       & \fun{groupByPool}~\var{dstr}~\var{delegs} = \\
-      & ~~~\Big\{ leader \mapsto \{hk \mapsto d \mid (hk \mapsto leader)\in\var{delegs}\}
-                  \Big\vert hk\mapsto d \in distr\Big\}
+      & ~~~\Big\{ leader \mapsto \{hk \mapsto d \in distr \mid (hk \mapsto leader)\in\var{delegs}\}
+                  \Big\}
```

Comment:
I think this needs to be a double comprehension, but here it is not clear what the outer set is.